pyproject.toml

```diff
@@ -8,34 +8,33 @@ packages = [{ include = "scripts" }]
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.10"
-cairo-lang = "^0.11.0.1"
+cairo-lang = "^0.11"
 openzeppelin-cairo-contracts = "0.5.0"
 amarna = "^0.1.5"
-starknet-py = "^0.15.0"
+starknet-py = "^0.15"
 
 [tool.poetry.group.dev.dependencies]
 black = "^22.10.0"
 isort = "^5.10.1"
-cairo-coverage = { git = "https://github.com/starknet-edu/cairo-coverage.git", branch="master" }
+cairo-coverage = { git = "https://github.com/starknet-edu/cairo-coverage.git", branch = "master" }
 pandas = "^1.5.1"
 pytest-xdist = { version = "^3.0.2", extras = ["psutil"] }
 setproctitle = "^1.3.2"
 pysha3 = "^1.0.2"
-web3 = "<6.0.0"
+web3 = "^6"
 python-dotenv = "^0.21.0"
-eth-account = "<0.8.0"
 autoflake = "^2.0.0"
 pycryptodome = "^3.16.0"
 rlp = "<=3"
-blake2b-py = { git = "https://github.com/ethereum/blake2b-py.git", tag="v0.2.0" }
-eth-keys = "<0.4.0"
-eth-abi = "<4.0.0"
-eth-utils = "<2.0.0"
 hypothesis = "^6.65.0"
 case-converter = "^1.1.0"
-starknet-devnet = "0.5.0a0"
+starknet-devnet = "0.5.0a2"
 requests = "^2.28.2"
 codespell = "^2.2.2"
+eth-account = "^0.8.0"
+eth-abi = "^4.0.0"
+eth-utils = "^2.1.0"
+eth-keys = "^0.4.0"
 
```

Comment:
I think you might be missing `py-evm` from your `project.toml` since you updated `test_blake2f.py` to reuse the previous method of testing (using `from eth._utils.blake2.compression`)